alerts/AggregatedAPIErrors.md

```diff
@@ -0,0 +1,65 @@
+# AggregatedAPIErrors
+
+## Meaning
+
+This alert is triggered in case multiple calls on the aggregated API of OpenShift failed over a certain time window.
+
+## Impact
+Errors on the aggregated API can result in the unavailability of some OpenShift services.  
+
+## Diagnosis
+
+##### Check the APIs status checks are on True
```

Comment:
nit: over-nested header.  Parent *Diagnosis* is an h2, so this should be `###`, not `#####`.